### PR TITLE
#600 Set-up analytics via Plausible

### PR DESCRIPTION
**Pull request by katarinazaprazna**

### CODE REVIEW

File: next/src/components/common/SeoHead.tsx [hunk 1]
```diff
@@ -44,9 +44,9 @@ const SeoHead = ({
 
   const fullPath = getFullPath(entity)
   const fullPathWithDomain = homepage
-    ? `https://olo.sk/`
+    ? `https://testing.bratislava.sk/`
     : fullPath
-      ? `https://olo.sk${fullPath}`
+      ? `https://testing.bratislava.sk${fullPath}`
```
Comment (radoslavzeman) [inline next/src/components/common/SeoHead.tsx:49]: Does Plausible care about it?

File: next/src/pages/_app.tsx [hunk 2]
```diff
@@ -27,29 +28,30 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
...
-            </OverlayProvider>
-            {/* </BAI18nProvider> */}
-          </QueryParamProvider>
-        </HydrationBoundary>
-      </BAQueryClientProvider>
-      {/* </PlausibleProvider> */}
+      <PlausibleProvider
+        domain="testing.bratislava.sk"
+        taggedEvents
+        enabled
```
Comment (radoslavzeman) [inline next/src/pages/_app.tsx:34]: What is this new prop? :) Isn't it default?

File: next/src/components/common/SeoHead.tsx (hunk 1 above)
Comment (katarinazaprazna) [reply next/src/components/common/SeoHead.tsx:49]: I had issues logging localhost events in Plausible, and this was one of the recommended temporary workarounds. It's not needed anymore and will be deleted in the next commit

File: next/src/pages/_app.tsx (hunk 2 above)
Comment (katarinazaprazna) [reply next/src/pages/_app.tsx:34]: The same as above. `enabled` is `true` by default :) A recommended workaround was to explicitly set the prop as `true`. It's not needed anymore and will be deleted in the next commit

**Pull request closed without merging**